packages/fxa-event-broker/src/app.module.ts

```diff
@@ -2,21 +2,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import { Module } from '@nestjs/common';
-import { ConfigModule } from '@nestjs/config';
+import { ConfigModule, ConfigService } from '@nestjs/config';
 import { ScheduleModule } from '@nestjs/schedule';
+import { HealthModule } from 'fxa-shared/nestjs/health/health.module';
+import { LoggerModule } from 'fxa-shared/nestjs/logger/logger.module';
+import { MetricsFactory } from 'fxa-shared/nestjs/metrics.service';
+import { SentryModule } from 'fxa-shared/nestjs/sentry/sentry.module';
+import { getVersionInfo } from 'fxa-shared/nestjs/version';
 
 import { AuthModule } from './auth/auth.module';
 import { ClientCapabilityModule } from './client-capability/client-capability.module';
 import { ClientWebhooksModule } from './client-webhooks/client-webhooks.module';
 import Config, { AppConfig } from './config';
 import { FirestoreModule } from './firestore/firestore.module';
-import { HealthModule } from './health/health.module';
 import { JwtsetModule } from './jwtset/jwtset.module';
-import { LoggerModule } from './logger/logger.module';
-import { MetricsFactory } from './metrics.service';
 import { PubsubProxyModule } from './pubsub-proxy/pubsub-proxy.module';
 import { QueueworkerModule } from './queueworker/queueworker.module';
-import { SentryModule } from './sentry/sentry.module';
+
+const version = getVersionInfo(__dirname);
```

Comment:
Woo! We now have a single function we can call from any package to get the version data. Looking forward to removing version.js/ts files all over.